pymeasure/instruments/activetechnologies/AWG401x.py

```diff
@@ -430,7 +430,8 @@ class AWG401x_AFG(AWG401x_base):
         print(wfg.check_errors())       # Get the error queue
 
     """
-    ch = Instrument.ChannelCreator(ChannelAFG, (1, 2))
+    ch_1 = Instrument.ChannelCreator(ChannelAFG, 1)
```

Comment:
This change will break the instrument, doesn't it?
If someone used "ch[1]", it won't work anymore.

Please keep it compatible, even if it breaks default pymeasure usage.
Maybe a workaround is "self.ch = self.channels" at the end of the "init".

Reply:
Yes, when fixing the tests I had the same thought. If the author wrote the tests with the collection, they probably are using it in the procedure. I have a further comment down below on this with regards to breaking changes.